confgenerator/testdata/valid/linux/metrics-receiver_redis_missing_address/golden_otel.conf

```diff
@@ -386,6 +386,12 @@ receivers:
     collection_interval: 30s
     endpoint: localhost:6379
     password: pwd
+    tls:
```

Comment:
Does OT treat all of these being set to "" the same as not specifying TLS at all?

Reply:
pretty much, it sets `insecure: true` as the default.